src/ofxGuiJson.cpp

```diff
@@ -0,0 +1,250 @@
+
+#include "ofxGuiJson.h"
+
+void ofxGuiJson::setup(const std::string& configFile) {
+    loadJson(configFile);
+    setBackgroundColor();  // Set background color based on JSON config
+    parseGuiElements();
+}
+
+void ofxGuiJson::loadJson(const std::string& relativePath) {
```

Comment:
I think there is some confusion with passed variable `relativePath`.
In setup, it looks like a "filename" is passed. But then setup simply calls `loadJson` and passes that name, which now becomes relative path?

Looks like the variable name in setup should cahnge from configFile to `path` and `ofToDataPath` should be moved to setup?

Let me know if I;m wrong in understanding this.